src/drivers/mct/cime_config/config_component.xml

```diff
@@ -2047,7 +2047,7 @@
     <default_value>0</default_value>
     <group>mach_pes_last</group>
     <file>env_mach_pes.xml</file>
-    <desc>total number of tasks and threads (setup automatically - DO NOT EDIT)</desc>
+    <desc>total number of MPI tasks (setup automatically - DO NOT EDIT)</desc>
```

Comment:
Now I'm confused - this should be the total number of physical cores used this would include threads unless the threads are using SMT.

Reply:
That is what I would have thought TOTALPES meant, but it has been returning number of MPI tasks recently.